### DISCUSSION TITLE
Drives table width too wide

### DISCUSSION THREAD

**Discussion by lq313c** [teslamate-org/teslamate#2818] [General]
Does anyone else find it annoying that the Drive table under in the Drives dashboard is too wide, and requires left/right scrolling to see all the columns? I think the Start/Destination columns are unnecessarily wide, and the min-width on those columns could be lowered to fit everything in the window.

**Comment by ktsenvolt** [comment]
I agree.

**Comment by MP3Mike** [comment]
Yes, I agree. Maybe it should be filed as a bug/issue.

**Comment by ktsenvolt** [comment]
![image](https://user-images.githubusercontent.com/80360674/193452259-90fdfece-1a2c-4bd1-9957-de287fec094c.png)
Is it possible to reduce the width of 'Vampire Drain ' column?
